hack/update-all.sh

```diff
@@ -90,4 +90,9 @@ do
 	fi
 done
 
-echo -e "${color_green}Update scripts completed successfully${color_norm}"
+if kube::util::ensure_clean_working_dir &>/dev/null; then
+    exec "${KUBE_ROOT}/hack/update-all-staging.sh" "$@"
```

Comment:
Would we need to do this for other scripts in the future (possibly)? Would it make sense to have a second array like `BASH_CLEAN_TARGETS` and iterate on that only if the working dir is clean?